app/scripts/controllers/users/edit.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name flightNodeDemo.controller:UserEditController
+ * @description
+ * # UserEditController
+ * Controller for the edit user page.
+ */
+angular.module('flightNodeDemo')
+	.controller('UserEditController', ['$scope', '$http', '$log', '$location', '$routeParams', 'messenger',
+		function ($scope, $http, $log, $location, $routeParams, messenger) {
+
+			$scope.loading = true;
+
+			var userId = $routeParams.userId;
+			if (!isFinite(userId)) {
+				// garbage input
+				return;
+			}
+
+			$scope.action = 'Edit';
+
+			var jwt = JSON.parse(localStorage.getItem('jwt'));
+
+			if (jwt) {
+				// TODO: validate that the user administration claim is present.
+				if (moment() < moment(jwt.expiresAt)) {
+
+					$http({
+						url: 'http://localhost:50323/api/v1/user/' + userId,
+						method: 'GET',
+						headers: {
+							Authorization: 'bearer ' + jwt.access_token
+						}
+					})
+						.then(function success(response) {
+
+							$scope.user = response.data;
+
+						}, function error(response) {
+
+							$log.error(response);
+
+							if (response.status === 401) {
+								messenger.unauthorized($scope);
+							} else {
+								messenger.showErrorMessage($scope, { error: response });
+							}
+						});
+				} else {
+					messenger.unauthorized($scope);
+				}
+			} else {
+				messenger.unauthorized($scope);
+			}
+
+			$scope.loading = false;
+
+			$scope.cancel = function () {
+				$location.path('/#/users');
+			}
+
+			$scope.submit = function () {
+				var jwt = JSON.parse(localStorage.getItem('jwt'));
+				
+				$log.info($scope);
+				
+				if (jwt) {
+					// TODO: validate that the user administration claim is present.
+					if (moment() < moment(jwt.expiresAt)) {
+						$http({
+							url: 'http://localhost:50323/api/v1/user/' + userId,
+							method: 'PUT',
+							data: $scope.user,
+							headers: {
+								Authorization: 'bearer ' + jwt.access_token
+							}
+						}).then(function success(response) {
+							messenger.showSuccessMessage($scope, 'Saved');
+						}, function error(response) {
+							$log.error(response);
+
+							switch (response.status) {
+								case 400:
+									var messages = [{ error: response.data.message }];
+
+									if (response.data.modelState) {
+										_.forIn(response.data.modelState, function (value, key) {
+											messages.push({ error: value.toString() });
+										});
+									}
```

Comment:
Actual sample response message from the Identity API service:

    
      "data":{
         "message":"The request is invalid.",
         "modelState"={
            "user.Email":[
               "The Email field is required."
            ],
            "user.UserName":[
               "The UserName field is required."
            ],
            "user.GivenName":[
               "The GivenName field is required."
            ],
            "user.FamilyName":[
               "The FamilyName field is required."
            ]
         }
      },
      "status":400,
      "config":{
         "method":"PUT",
         "transformRequest":[
            null
         ],
         "transformResponse":[
            null
         ],
         "url":"http://localhost:50323/api/v1/user/1",
         "data":{
            "id":"1",
            "user":{
               "userId":1,
               "email":"ab@asfddsdfs.com",
               "primaryPhoneNumber":"555-555-5555",
               "userName":"asdf",
               "secondaryPhoneNumber":null,
               "givenName":"Juana",
               "familyName":"Coneja",
               "password":""
            }
         },
         "headers":{
            "Authorization":"bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJuYW1laWQiOiIxIiwidW5pcXVlX25hbWUiOiJhc2RmIiwiaHR0cDovL3NjaGVtYXMubWljcm9zb2Z0LmNvbS9hY2Nlc3Njb250cm9sc2VydmljZS8yMDEwLzA3L2NsYWltcy9pZGVudGl0eXByb3ZpZGVyIjoiQVNQLk5FVCBJZGVudGl0eSIsIkFzcE5ldC5JZGVudGl0eS5TZWN1cml0eVN0YW1wIjoiOGRkZmQ0YTctYTEzZi00ZmQwLWFkMDItNWI4ZWQ2NTYwOTdjIiwiaXNzIjoiaHR0cDovL2xvY2FsaG9zdDo1MDMyMyIsImF1ZCI6Ikp1c3RNYWtldXBTb21ldGhpbmciLCJleHAiOjE0NDUzMDg5ODQsIm5iZiI6MTQ0NTIyMjU4NH0.zoQlTBv0JOs__S_hcenffEUPhB7MBDWU2mSWf128D38",
            "Accept":"application/json, text/plain, */*",
            "Content-Type":"application/json;charset=utf-8"
         }
      },
      "statusText":"Bad Request"